addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -2090,6 +2091,19 @@ var SnippetsMenu = Widget.extend({
             }],
         }).open();
     },
+    /**
+     * @private
+     */
+    _onMouseDown: function () {
+        $('#wrapwrap').addClass('disable_pointer_events');
+        const enablePointerEvents = setTimeout(function () {
+            $('#wrapwrap').removeClass('disable_pointer_events');
+        }, 2000);
```

Comment:
Increase to 5 seconds to be sure but also add a comment to explain why we do that